Taarafo.Core.Tests.Acceptance/Apis/Comments/CommentsApiTests.Logic.cs

```diff
@@ -44,11 +44,17 @@ public async Task ShouldGetAllCommentsAsync()
             List<Comment> actualComments = await this.apiBroker.GetAllCommentsAsync();
 
             // then
+            actualComments.Count.Should().BeGreaterThanOrEqualTo(expectedComments.Count);
+            actualComments.Count.Should().BeLessThanOrEqualTo(10);
+
             foreach (Comment expectedComment in expectedComments)
             {
-                Comment actualComment = actualComments.Single(comment => comment.Id == expectedComment.Id);
-                actualComment.Should().BeEquivalentTo(expectedComment);
-                await this.apiBroker.DeleteCommentByIdAsync(actualComment.Id);
+                if (actualComments.Any(comment => comment.Id == expectedComment.Id))
```

Comment:
@hassanhabib is this acceptable for the acceptance test?  Due to the page size, the new records might not come up in page one. So I did the test to check the page size apply and that if no data exists, the results is at least meeting the count of the list.

Reply:
@cjdutoit hold off on this odata configuration until we are ready to go to production please